tests/vpp.cpp

```diff
@@ -29,6 +29,11 @@
 
 using namespace YamiMediaCodec;
 
+#define HUE_DEFAULT 50
```

Comment:
why we use this as default value, is it special? what if I just want set hue to 50?
Could we just use COLORBALANCE_NONE etc.